docs/docs/event-history/README.md

```diff
@@ -107,8 +107,14 @@ You'll need to replay an event to execute the workflow with the latest changes.
 
 ::: tip Beta access
 
-During the beta, paid accounts will have access to view the past 7 days of events across all of their workflows. This limit will increase as this trial continues.
+During the beta, all accounts will have access to view the past 7 days of events across all of their workflows. This limit will increase as this trial continues.
 
 :::
 
 The number of events recorded and available for viewing in the Event History depends on your plan. [Please see the pricing page](https://pipedream.com/pricing) for more details.
+
+## Frequently Asked Questions
+
+### Are event histories available on all plans?
```

Comment:
```suggestion
### Are event histories available on all plans?
```

Reply:
Changed "histories" to lowercase to follow Google's style guide on capitalization.